packages/schemas/src/gen/index.ts

```diff
@@ -83,6 +83,8 @@ const generate = async () => {
                 return {
                   name,
                   type: primitiveType,
+                  isString,
```

Comment:
nit: could we simply use `type === string` here?

Reply:
<img width="694" alt="image" src="https://user-images.githubusercontent.com/10594507/183559064-9bc1cf76-16e4-436b-acbb-084ad9177dff.png">

Since `isString` is also used in `parseType` util function, I reuse `isString` returned by `parseType` for now.